docs/db/changelog/changesets/powerauth-java-server/1.6.x/20231106-add-foreign-keys.xml

```diff
@@ -4,7 +4,7 @@
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
 
     <changeSet id="1" logicalFilePath="powerauth-java-server/1.6.x/20231106-add-foreign-keys.xml" author="Jan Pesek">
-        <preConditions onFail="MARK_RAN">
+        <preConditions onFail="MARK_RAN" onError="MARK_RAN">
```

Comment:
If this doesn't change the checksum - we can use it like this.

Reply:
Indeed, it does not affect the checksum.